js/lesson4/tutorial.md

```diff
@@ -104,7 +104,7 @@ The following code allows you to listen for a keypress on the input field, and t
 
 ```js
 $(document).ready(function(){
-  $(document).on('keypress', '#username', function(e){
+  $(document).on('keypress', '#username', function(){
```

Comment:
Yep I've just tested it. The `e` is not needed - `event` is given to you somehow.